libs/template/templates/default-python/template/{{.project_name}}/databricks.yml.tmpl

```diff
@@ -0,0 +1,58 @@
+# This is a Databricks Asset Bundle definition for {{.project_name}}.
+# See https://docs.databricks.com/en/dev-tools/bundles for documentation.
+bundle:
+  name: {{.project_name}}
+
+include:
+  - resources/*.yml
+
+artifacts:
+  {{.project_name}}:
+    type: whl
+    path: "."
+    build: "python3 setup.py bdist_wheel"
+
+targets:
+  # The 'dev' target, used development purposes.
+  # Whenever a developer deploys using 'dev', they get their own copy.
+  dev:
+    # We use 'mode: development' to make everything deployed to this target gets a prefix
+    # like '[dev my_user_name]'. Setting this mode also disables any schedules and
+    # automatic triggers for jobs and enables the 'development' mode for DLT pipelines.
+    mode: development
+    default: true
+    workspace:
+      host: {{workspace_host}}
+
+  # Optionally, there could be a 'staging' target here.
+  # (See Databricks docs on CI/CD at https://docs.databricks.com/en/dev-tools/bundles.)
+  #
+  # staging:
+  #  workspace:
+  #    host: {{workspace_host}}
+
+  # The 'prod' target, used for production deployment.
+  prod:
+    # For production deployments, we only have a single copy, so we override the
+    # workspace.root_path default of
+    # /Users/${workspace.current_user.userName}/.bundle/${bundle.environment}/${bundle.name}
+    # to a path that is not specific to the current user.
+    {{- /*
+    Explaining 'mode: production' isn't as pressing as explaining 'mode: development'.
+    As we already talked about the other mode above, users can just
+    look at documentation or ask the assistant about 'mode: production'.
+    #
+    # By making use of 'mode: production' we enable strict checks
+    # to make sure we have correctly configured this target.
+    */}}
+    mode: production
+    workspace:
+      host: {{workspace_host}}
+      root_path: /Shared/.bundle/prod/${bundle.name}
+    {{- if not is_service_principal}}
+    run_as:
+      # This runs as {{user_name}} in production. Alternatively,
+      # a service principal could be used here using service_principal_name
+      # (see Databricks documentation).
```

Comment:
"see Databricks documentation..." include a direct link?

Reply:
I wasn't sure what to link to here. I don't think we have a page that explains how to use the `service_principal_name` feature of DABs that also explains service principals. Such a page could describe that you would need to put the 'id' of the principal there and where to find that id.

I could just leave the text as seen above and link to https://docs.databricks.com/administration-guide/users-groups/service-principals.html?

Reply:
Probably best to just leave as is.